data/json/monsters/power_leech.json

```diff
@@ -0,0 +1,188 @@
+[
+  {
+    "id": "mon_leech_blossom",
+    "type": "MONSTER",
+    "name": "leech blossom",
+    "description": "A resplendent alien fern, crowned with flowers colored deep indigo.  It appears to be the centerpiece of this otherworldly bloom.",
+    "default_faction": "nether",
+    "species": [ "LEECH_PLANT" ],
+    "volume": "92500 ml",
+    "weight": "40 kg",
+    "hp": 100,
+    "speed": 100,
+    "material": [ "flesh" ],
+    "symbol": "K",
+    "color": "light_cyan",
+    "aggression": 100,
+    "morale": 100,
+    "armor_bash": 15,
+    "vision_day": 30,
+    "vision_night": 30,
+    "luminance": 200,
+    "special_attacks": [
+      {
+        "type": "gun",
+        "cooldown": 5,
+        "gun_type": "emp_frond",
+        "fake_skills": [ [ "gun", 3 ], [ "rifle", 3 ] ],
+        "ranges": [ [ 0, 12, "DEFAULT" ] ],
+        "description": "Lightning arcs towards you!"
+      },
+      [ "LEECH_SPAWNER", 25 ],
+      [ "MON_LEECH_EVOLUTION", 40 ],
+      [ "PARROT", 40 ]
+    ],
+    "death_drops": {  },
+    "death_function": [ "NORMAL" ],
+    "flags": [ "SEES", "NOHEAD", "IMMOBILE", "NO_BREATHE", "QUEEN", "HARDTOSHOOT" ]
+  },
+  {
+    "id": "mon_leech_stalk",
+    "type": "MONSTER",
+    "name": "leech stalk",
+    "description": "A resplendent and voluminous alien fern. A faint buzzing sound emanates from it, and the shadow cast by its canopy continuously glows with electric charge.",
+    "default_faction": "nether",
+    "species": [ "LEECH_PLANT" ],
+    "volume": "30000 ml",
+    "weight": "40 kg",
+    "hp": 40,
+    "speed": 100,
+    "material": [ "flesh" ],
+    "symbol": "y",
+    "color": "light_blue",
+    "aggression": 100,
+    "morale": 100,
+    "armor_bash": 15,
+    "vision_day": 30,
+    "vision_night": 30,
+    "luminance": 200,
+    "special_attacks": [
+      {
+        "type": "gun",
+        "cooldown": 15,
+        "gun_type": "emp_frond",
+        "fake_skills": [ [ "gun", 2 ], [ "rifle", 2 ] ],
+        "ranges": [ [ 0, 12, "DEFAULT" ] ],
+        "description": "Lightning arcs towards you!"
+      },
+      [ "MON_LEECH_EVOLUTION", 30 ],
+      [ "PARROT", 40 ]
+    ],
+    "death_drops": {  },
+    "death_function": [ "NORMAL" ],
+    "flags": [ "SEES", "NOHEAD", "ELECTRONIC", "IMMOBILE", "NO_BREATHE", "HARDTOSHOOT" ]
```

Comment:
`ELECTRONIC` flag here will result in all sorts of weird behavior, like showing a message of `The root drone sparks and begins searching for a target!` or `The EMP blast fries the root drone!` on using scramblers and emp grenades on it; you'll be able to use robot controls on it; and others. Here and most other monsters.

Reply:
Robot control works based on robot species,  and they being weak to emp is intended, but you are right about the scrambler grenade, I'll change it.